<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesWindow</class>
 <widget class="QDialog" name="NotesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ButtonNew">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>1281</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSaveED">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>580</y>
     <width>541</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить/(расшифровать/расшифровать)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonMenu">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>610</y>
     <width>541</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>В главное меню</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLoadFile">
   <property name="geometry">
    <rect>
     <x>744</x>
     <y>610</y>
     <width>541</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSaveFile">
   <property name="geometry">
    <rect>
     <x>744</x>
     <y>580</y>
     <width>541</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить в новый файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonRemove">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>610</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>580</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Слово</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
